cmd/awscollector/Dockerfile

```diff
@@ -41,10 +42,12 @@ FROM scratch
 COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 COPY --from=build /workspace/build/linux/aoc_linux_x86_64 /awscollector
 COPY --from=build /workspace/config.yaml /etc/otel-config.yaml
+COPY --from=build /workspace/config/ecs/container-insights/otel-agent-default-config.yaml /etc/ecs/container-insights/otel-agent-default-config.yaml
+COPY --from=build /workspace/config/ecs/otel-agent-ecs-default-config.yaml /etc/ecs/otel-agent-ecs-default-config.yaml
```

Comment:
I suggest we should remove `otel-agent` from config file name. Can we make the file name more meaningful?
* ecs-default-config.yaml
* ecs-container-metrics-config.yaml

Reply:
👍

Reply:
How about:
- /etc/ecs/ecs-default-config.yaml
- /etc/ecs/container-insights/otel-task-metrics-config.yaml

Since the container insights config gives you task level metrics..